modules/localgov_admin_role/localgov_admin_role.install

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the localgov_admin_role module.
+ */
+
+use Drupal\user\Entity\Role;
+
+/**
+ * Implements hook_install().
+ */
+function localgov_admin_role_install() {
+  // Check if the 'localgov_admin_role' role exists.
+  $admin_role = Role::load('localgov_admin');
```

Comment:
I suggest replacing the string by a class constant in the RolesHelper class, as this is what we're doing with other roles and makes the constant available should we wish to do any manipulation of this role later on.

https://github.com/localgovdrupal/localgov_core/blob/1e3c35bc65b1c97fae390cdaeccf45a0eb6f74fc/modules/localgov_roles/src/RolesHelper.php#L8-L28